### DISCUSSION TITLE
Split "rideable" into "rideable" and "controllable"

### DISCUSSION THREAD

**Discussion by 4P5** [PurpurMC/Purpur#442] [__Completed__] [answered]
## Is your feature request related to a problem?

Currently riding a mob and piloting a mob are under the same setting; `ridable: true`. I'd like to let players ride mobs like phantoms and dragons, but not to control them, to add some unique combat.

(While we're on the topic of hostile mobs, perhaps a "damage-amount" and "damage-rate" option for riding them? Defaults to 0/0, but would stop players from just riding a wither for an hour, with something like 2/1)

## Describe the solution you'd like.

For the default config, something like `rideable: false`, with the additional of `controllable`, which defaults to `true` (for compatibility), and is dependant on rideable. This won't break any previous versions, but admins can then disable the control feature and only enable riding.

**Comment by BillyGalbreath** [comment, marked as answer]
Implemented in build 1519

https://github.com/PurpurMC/Purpur/commit/bd0a85a17c9a9ffcb3ec244ebd1eacd517ae2956
